src/api/app/assets/stylesheets/webui/user_profile_redesign/user_profile.scss

```diff
@@ -0,0 +1,5 @@
+// TODO: As soon as user-profile-redesign feature is 100% rolled out,
+// all the 'body.user-profile-redesign' wrappers can be removed.
+
+body.user-profile-redesign  {
+}
```

Comment:
add a new line, please

Reply:
Done. But I've taken out the corresponding commit, I'll add it to the next PR.